config/locales/cy.yml

```diff
@@ -4,7 +4,7 @@ cy:
   common:
     last_updated: "Diweddarwyd diwethaf"
   components:
-    radio:
+    radio: # TODO: Push this upstream to the govuk_publishing_gem and remove from here
```

Comment:
See this TODO, I made a mistake forgetting to move the Welsh translation of the Radio component when moving it into the gem.

I'll follow this up with another PR, but this is a good illustration of why co-locating file is so important, this would have been pretty hard to mess up if the files were right next to each other.